include/torch-mlir/Dialect/TorchConversion/Transforms/Passes.h

```diff
@@ -28,7 +28,19 @@ void createTorchBackendToLinalgOnTensorsBackendPipeline(OpPassManager &pm);
 
 /// Creates a pipeline that lowers from the torch backend contract to the
 /// TOSA backend contract.
-void createTorchBackendToTosaBackendPipeline(OpPassManager &pm);
+struct TosaBackendPipelineOptions
+    : public PassPipelineOptions<TosaBackendPipelineOptions> {
+  // If this option is true, custom complex operations.
+  // If this option is false, skip decomposition of complex operations.
+  Option<bool> enableCustomOps{*this, "enableCustomOps",
```

Comment:
I don't see any other reference to this, is this option needed, or is the custom-ops ListOption sufficient?

Reply:
Deleted